src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -873,7 +873,7 @@ def test_edit_media_uploadedicon(self):
 
     def test_edit_media_icon_log(self):
         self.test_edit_media_uploadedicon()
-        log = ActivityLog.objects.all()
+        log = ActivityLog.objects.exclude(action=amo.LOG.LOG_IN.id)
```

Comment:
if we have to do this a lot maybe we need a manger/queryset shortcut for objects without `LOG_IN`?

Reply:
also, `LOG.LOG_IN` 😬

Reply:
IMHO it'd not worth having a shortcut for it - We could make each of those assertions look for specific actions instead of excluding but I didn't have the motivation to do that...